test/franklin_web/admin-user-stories/can_create_and_edit_with_article_editor_test.exs

```diff
@@ -0,0 +1,189 @@
+defmodule FranklinWeb.AdminUserStories.CanCreateAndEditWithArticleEditor do
+  @moduledoc """
+  Asserts the business rule that an authenticated admin can create or edit an
+  article and that field value failures works as expected.
+  """
+
+  use FranklinWeb.ConnCase
+
+  alias Franklin.Articles
+  alias Franklin.Articles.Article
+
+  setup %{conn: conn} do
+    # Because the article editor is used for both the creation and editing of
+    # articles, we'll construct two instances of the live view, allowing us to
+    # shape the tests to verify aspects of the form for both its create version
+    # as well as its edit version.
+
+    conn = add_authentication(conn, "zorn", "Pass1234")
+    {:ok, create_view, _html} = live(conn, "/admin/articles/editor/new")
+
+    now = DateTime.utc_now() |> DateTime.truncate(:second)
+
+    # This will be the article we will edit.
+    edit_article_attributes = %{
+      title: "edit article title",
+      body: "edit article body",
+      published_at: now
+    }
+
+    {:ok, edit_article_id} = Articles.create_article(edit_article_attributes)
+
+    edit_article =
+      wait_for_passing(fn ->
+        # Because projections are not instant, we need to wait until it is finished.
+        assert %Article{
+                 title: "edit article title",
+                 body: "edit article body",
+                 published_at: ^now
+               } = Articles.get_article(edit_article_id)
+      end)
+
+    {:ok, edit_view, _html} = live(conn, "/admin/articles/editor/#{edit_article.id}")
+
+    ~M{create_view, edit_view, edit_article}
```

Comment:
This sytax `~M{create_view, edit_view, edit_article}` comes from https://github.com/meyercm/shorter_maps.

I discovered this on a previous client project. Initially did not like it, but in test code, this map repetition is so common I thought I'd pull it in.